data/en/millisecond.json

```diff
@@ -5,14 +5,14 @@
 	"member": "date.millisecond([timezone])",
 	"returns": "numeric",
 	"related": [],
-	"description": "",
+	"description": "Returns the milliseconds of the referenced date.",
 	"params": [
-		{"name": "date", "description": "", "required": true, "default": "", "type": "date", "values": []},
-		{"name": "timezone", "description": "", "required": false, "default": "", "type": "string", "values": []}
+		{"name": "date", "description": "date object; for example, now()", "required": true, "default": "", "type": "date", "values": []},
```

Comment:
I'm not sure if providing an example is really needed in this place…

Reply:
Probably not necessary, but it's not too wordy so I'm fine with it.